tests/mr/device/arena_mr_tests.cpp

```diff
@@ -15,6 +15,7 @@
  */
 
 #include "../../byte_literals.hpp"
+#include "cuda/stream_ref"
```

Comment:
```suggestion
#include <cuda/stream_ref>
```

And subsequent include reordering?

Reply:
Yep, blame clangd.  Good catch.

Reply:
Fixed.